src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,75 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { getNumbers } from '../../utils';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: React.Dispatch<number>;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage = 1,
```

Comment:
The default value for `currentPage` is set inside the destructuring of props. This is not a common practice in TypeScript because the default props should be defined in a separate static property `defaultProps` or by using a default function parameter.